### PR TITLE
updated cmake

### PR DESCRIPTION
**Pull request by mdering**
updated cmake to version 3.2.2

### CODE REVIEW

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

Specific versions of apps should be submitted to [caskroom/versions](https://github.com/caskroom/homebrew-versions).

File: Casks/cmake.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'cmake' do
-  version '3.1.1'
-  sha256 '5106e940d8abcdbd2c8b5117ee9618c90207aeb4235837e5170aaf1a950230f8'
+  version '3.2.2'
+  sha256 '430b9f94a363ff19d5f66eacc5d6bc55bd9dac75d6fc6bcb813d8b451c1475c4'
 
-  url "http://www.cmake.org/files/v3.1/cmake-#{version}-Darwin-x86_64.dmg"
+  url "http://www.cmake.org/files/v3.2/cmake-#{version}-Darwin-x86_64.dmg"
```
Comment (vitorgalvao) [inline Casks/cmake.rb]: This should be changed to `url "http://www.cmake.org/files/v#{version.sub(%r{\.\d+},''}/cmake-#{version}-Darwin-x86_64.dmg"`.

Comment (mdering) [inline Casks/cmake.rb]: fixed!

**Comment by vitorgalvao** [comment]
`weka37` still needs to be removed. Please [squash your commits](http://davidwalsh.name/squash-commits-git) after.

**Comment by mdering** [comment]
I may have done something wrong because it doesnt look like I can squash my commits, but anyway, i removed weka, let me know if i should just revert and start over instead. Sorry about that.

**Comment by vitorgalvao** [comment]
No problem. I’ll squash them for you in the below referenced pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes. Thank you for the contribution.

**Pull request closed without merging**